src/goto-symex/symex_function.cpp

```diff
@@ -117,7 +117,14 @@ unsigned goto_symext::argument_assignments(
 
       // Assign value to function argument
       // TODO: Should we hide it (true means hidden)?
-      symex_assign(code_assign2tc(lhs, rhs), true);
+      if (options.get_bool_option("generate-html-report"))
```

Comment:
Can you do `symex_assign(code_assign2tc(lhs, rhs), !options.get_bool_option("generate-html-report"));`?

Also please add a comment to explain why we are doing this

Reply:
Yes, this is in response to a previous comment where you said that the change in symex_assign should only be applied if a html report is being generated.

Reply:
I know, but the previous version of the code with if/else was not good.

Please also adjust the comment there too